plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteClientModule.java

```diff
@@ -53,13 +52,10 @@ public void configure(Binder binder)
     @ForBaseJdbc
     public static ConnectionFactory createConnectionFactory(BaseJdbcConfig config, CredentialProvider credentialProvider)
     {
-        // https://github.com/trinodb/trino/issues/16148
-        checkArgument(requireNonNull(config.getConnectionUrl()).startsWith(JdbcThinUtils.URL_PREFIX), "Invalid JDBC URL for Ignite connector, connection url should start with 'jdbc:ignite:thin://', actual get %s", config.getConnectionUrl());
-        Properties connectionProperties = new Properties();
         return new DriverConnectionFactory(
                 new IgniteJdbcThinDriver(),
                 config.getConnectionUrl(),
-                connectionProperties,
+                new Properties(),
```

Comment:
We can change `config.getConnectionUrl()` to `config` and then remove this `new Properties()` argument.